includes/application.php

```diff
@@ -33,6 +33,24 @@
     $app->bind(HandlerInterface::class, 'error.handler.development');
     ini_set('display_errors', true);
     error_reporting(E_ALL);
+
```

Comment:
I would definitely move this to a `VarDumpServerServiceProvider` to keep this legacy and bootstrapping code clean

Reply:
sure, will refactor this